### ISSUE TITLE
pre-Mad-X-run file checks

### ISSUE THREAD

**Issue by JoschD** [pylhc/Beta-Beat.src#89]
Before calling MadX check that all required files and the output directory exist (at least from job.twiss.madx and the optics-file)

**Comment by jaimecp89** [comment]
You mean for the model creator? Or in the madx_runner itself?

**Comment by JoschD** [comment]
Both, actually. The madx runner has only an input file or text which may calls some files whos existence could be checked., the creator knows which one of these is the optics file and can check everything in there... like one level deeper.
A simple parser, called in both cases with the madx-string/content of the file, looking for the paths that follow "call, file" should do the trick.

Why? I did not notice that I had afs not mounted and the first error I got was way way after these functions when some other method complained that it couldn't find the model (maybe a good final sanity check for the creator as well...) 
[actually, it was way worse, as also the other madx commands for creating the fullresponse, which failed, did not complain either...]
